scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -200,20 +202,78 @@ def configure_machine_id(self, phase):
     else:
         raise MotrError(errno.ENOENT, "machine id not available in conf")
 
-def get_server_node(self, k8):
+def get_server_node(self):
     """Get current node name using machine-id."""
     try:
         machine_id = self.machine_id;
-        if k8:
-            server_node = Conf.get(self._index, 'node')[machine_id]
-        else:
-            server_node = Conf.get(self._index, 'server_node')[machine_id]
+        server_node = Conf.get(self._index, 'node')[machine_id]
     except:
         raise MotrError(errno.EINVAL, f"MACHINE_ID {machine_id} does not exist in ConfStore")
 
     check_type(server_node, dict, "server_node")
     return server_node
 
+def calc_size(self, sz):
+    ret = -1
+    suffixes = ['K', 'Ki', 'Kib', 'M', 'Mi', 'Mib', 'G', 'Gi', 'Gib']
+    sz_map = {
+              "K": 1024, "M": 1024*1024, "G": 1024*1024*1024,
+              "Ki": 1024, "Mi": 1024*1024, "Gi": 1024*1024*1024,
+              "Kib": 1024, "Mib": 1024*1024, "Gib": 1024*1024*1024 }
+
+    # Check if sz ends with proper suffixes. It matches only one suffix.
+    temp = list(filter(sz.endswith, suffixes))
+    if len(temp) > 0:
+        suffix = temp[0]
+        num_sz = re.sub(r'[^0-9]', '', sz) # Ex: If sz is 128MiB then num_sz=128
+        map_val = sz_map[suffix] # Ex: If sz is 128MiB then map_val = 1024*1024*1024
+        ret = int(num_sz) * int(map_val)
+        return ret
+    else:
+        self.logger.error(f"Invalid format of mem limit: {sz}\n")
+        self.logger.error("Please use valid format Ex: 1024, 1Ki, 1Mi, 1Gi etc..\n")
+        return ret
+
+def get_setup_size(self, service):
+    ret = False
+    sevices_limits = Conf.get(self._index, 'cortx>motr>limits')['services']
+
+    #Provisioner passes io as parameter to motr_setup.
+    #Ex: /opt/seagate/cortx/motr/bin/motr_setup config --config yaml:///etc/cortx/cluster.conf --services io
+    #But in /etc/cortx/cluster.conf io is represented by ios. So first get the service names right
+    if service == "io":
+         svc = "ios"
+    else:
+         svc = service
```

Comment:
do we need this in " "

Reply:
service is the function param so not needed in ""